src/Aquifer.Public.API/Endpoints/Resources/Collections/Get/Endpoint.cs

```diff
@@ -0,0 +1,211 @@
+using System.Data.Common;
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Public.API.Helpers;
+using Dapper;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Collections.Get;
+
+public sealed class Endpoint(AquiferDbContext _dbContext, ICachingLanguageService _cachingLanguageService)
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Get("/resources/collections/{code}");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+        Summary(s =>
+        {
+            s.Summary = "Gets a resource collection with language localization data for the given collection code.";
+            s.Description =
+                "Returns the resource collection matching the collection code including the count of resource items in each available language.";
+        });
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
```

Comment:
I'm happy with the current performance and long-term sustainability of the performance of this route.  Also, if anything gets expensive we could add caching of the counts.

As it stands right now fetching the counts for all languages for a single parent resource doesn't take longer than ~200ms and the other two queries are fast.

Getting it fast required moving away from the old query used by the Internal API.  See below for more comments about this.